DotNET/Complex Flow Examples/decrypt-add-reencrypt.cs

```diff
@@ -0,0 +1,98 @@
+using Newtonsoft.Json.Linq;
+using System.Text;
+
+/* In this sample we will show how to take an encrypted file and decrypt, modify
+* and re-encrypt it to create an encryption-at-rest solution as discussed in
+* https://pdfrest.com/solutions/create-secure-document-workflows-with-pdf-password-protection/
+* We will be running the document through /decrypted-pdf to open the document
+* to modification, running the decrypted result through /pdf-with-added-image,
+* and then sending the output with the new image through /encrypted-pdf to
+* lock it up again.
+*/
```

Comment:
```suggestion
/* In this sample, we will show how to take an encrypted file and decrypt, modify
* and re-encrypt it to create an encryption-at-rest solution as discussed in
* https://pdfrest.com/solutions/create-secure-document-workflows-with-pdf-password-protection/
* We will be running the document through /decrypted-pdf to open the document
* to modification, running the decrypted result through /pdf-with-added-image,
* and then sending the output with the new image through /encrypted-pdf to
* lock it up again.
*/
```

Just tossed a comma in